modules/hal_nordic/nrfx/nrfx_glue.h

```diff
@@ -275,8 +275,10 @@ void nrfx_busy_wait(uint32_t usec_to_wait);
 #define NRFX_GPIOTE_CHANNELS_USED NRFX_GPIOTE_CHANNELS_USED_BY_BT_CTLR
 
 #if defined(CONFIG_BT_CTLR)
-#include <../subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_resources.h>
-
+/* The file to be included here needs to be specified in cmake definitions
+ * of the Bluetooth controller subsystem to be used.
+ */
+#include BT_CTLR_USED_RESOURCES
```

Comment:
Including a define looks a bit strange. By naming it `BT_CTLR_USED_RESOURCES_INCLUDE_FILE` it becomes more obvious that is really intentional. 

I'm open for other suggestions as well

Reply:
Good point. I'll update the name.